concent_api/api-integration-multi-threads-test.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+import hashlib
+import os
+import sys
+import time
+from datetime import datetime
+from threading import Thread
+
+import requests
+from golem_messages.shortcuts import dump
+
+from api_testing_common import api_request
+from api_testing_common import assert_condition
+from api_testing_common import count_fails
+from api_testing_common import get_force_get_task_result
+from api_testing_common import REQUESTOR_PRIVATE_KEY
+from api_testing_common import run_tests
+from common.helpers import get_current_utc_timestamp
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concent_api.settings")
+
+now = datetime.now()
+testing_subtask_id = str(now.year) + str(now.month) + str(now.day) + str(now.hour) + str(now.minute) + str(now.second) + str(now.microsecond)
```

Comment:
Maybe it is good time to start using UUID as ours id's?

Reply:
I have actually skipped the context of this line, so yes, generating `subtask_id` should look much simpler and what @kbeker proposed is good idea.

Reply:
@rwrzesien maybe we should close this issue as fast as it possible and open new issue with changing all places where `subtask_id` and `task_id` is created to UUID?

Reply:
I've just add issue with adding to our code UUID id's: https://github.com/golemfactory/concent/issues/705

Reply:
Works for me.